package.json

```diff
@@ -120,5 +120,8 @@
 		"tabWidth": 2,
 		"trailingComma": "es5",
 		"useTabs": true
+	},
```

Comment:
My only question about this feature is that the js client no longer uses external libraries. Community asked to remove and we did. Please check with the decision makers for this project to give you their green light before you release this version.

Reply:
@BibiSebi As Thiago said, the focus of `storyblok-js-client` is to have no external dependencies in order to be as lightweight as possible. What do you think ?